models/basgra/DESCRIPTION

```diff
@@ -0,0 +1,19 @@
+Package: PEcAn.BASGRA
+Type: Package
+Title: PEcAn package for integration of the BASGRA model
+Version: 1.7.1
+Date: 2019-09-05
+Authors@R: c(person("Istem", "Fer", email = "istem.fer@fmi.fi", role = c("aut", "cre")))
+Description: This module provides functions to link the BASGRA model to PEcAn.
+Imports:
+    PEcAn.logger,
+    PEcAn.utils (>= 1.4.8)
```

Comment:
You should add other dependent packages here: `lubridate`, `ncdf4`, `udunits2`, and any others I may have missed.